src/components/Text/index.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/require-default-props */
+import React from 'react';
+
+import { Container, TextStyleProps } from './styles';
+
+export interface TextProps extends TextStyleProps {
+  className?: string;
+  forwardRef?: React.RefObject<HTMLDivElement>;
+  noSelect?: boolean;
+}
+
+const Text: React.FC<TextProps> = ({
+  className,
+  forwardRef,
+  noSelect,
+  inline,
+  fontFamily,
+  fontSize,
+  color,
+  children,
+  underlined,
+}) => {
+  return (
+    <Container
+      className={`text${noSelect ? ' no-user-select' : ''}${
+        className ? ` ${className}` : ''
+      }`}
```

Comment:
This will only work properly after setting the `className` default prop as `''`.

```suggestion
      className={`epg-text ${className} ${noSelect ? ' no-user-select' : ''}`}
```